src/App.js

```diff
@@ -1,13 +1,21 @@
-import React from 'react';
-import { Route, Switch } from 'react-router-dom';
+import React, { useState } from 'react';
+import { Route, Switch, Redirect } from 'react-router-dom';
 import './App.css';
-import Test from './components/Test';
 import Navigation from './components/Navigation';
 import AddItem from './components/AddItem';
 import List from './components/List';
 import Welcome from './components/Welcome';
+import getToken from './lib/tokens';
 
 function App() {
+  const [token, setToken] = useState(localStorage.getItem('userToken') || '');
```

Comment:
impressive use of the logical || operator!

Reply:
Thanks for the feedback! Epicodus had us use PropTypes, but I haven't seen them used in some of the recent tutorials I've watched, so it's good to get a reminder on why they're helpful! And thank you for linking to some additional resources for further learning 👍

Reply:
In my opinion, the #1 reason to use prop-types is for documentation purposes. It's always really useful when I go into a component and see something that looks like:

```
Component.propTypes = {
   // Super useful comment explaining the exact purpose of the prop
   onClick: PropTypes.func.isRequired
}
```
Whenever I see well documented prop-types, I usually know that it was written by someone who was really trying to be helpful to other devs :-)